CHANGELOG.md

```diff
@@ -21,8 +21,30 @@ Deprecated modules
 Deprecated names
 ----------------
 
+* In `Data.List.Base`:
+  ```agda
+  sum       ↦  Data.Nat.ListAction.sum
+  product   ↦  Data.Nat.ListAction.product
+  ```
+
+* In `Data.List.Properties`:
+  ```agda
+  sum-++       ↦  Data.Nat.ListAction.sum-++
```

Comment:
Hmm, I think these ought to be in `Data.Nat.ListAction.Properties`. I still will want to have `sum` and `product` available with a decently small footprint (dependency-wise) and bundling properties with definitions defeats that.

Reply:
And a corresponding `Base` module for the definitions?